app/assets/stylesheets/apply.scss

```diff
@@ -67,6 +67,39 @@
   z-index: -1;
 }
 
+.authorize-icon {
+  font-size: 3em;
+  position: absolute;
+}
+
+.fb-authorize-icon {
+  bottom: 156px;
+  left: 39px;
+}
+
+.google-authorize-icon {
+  bottom: 68px;
+  left: 39px;
+}
+.fb-authorize-button {
+  width: 100%;
+  background: #3b5998;
+}
+
+
+.fb-authorize-button:hover {
+  background: #2f4779;
+}
+
+.google-authorize-button {
+  width: 100%;
+  background: #dd4b39;
+}
+
+
+.google-authorize-button:hover {
+  background: #b03c2d;
```

Comment:
Color literals like `#b03c2d` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#b03c2d` should be written as `#B03C2D`